WebClient/src/components/details/itemsContainer.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * ItemsContainer is a generic container to store model and their views.
```

Comment:
Maybe add a reference that ItemsContainer contains Detail item that contain model and view